_data/schedule.yml

```diff
@@ -1,164 +1,154 @@
 - 
-  date: "Aug 27"
+  date: "Jan 14"
   materials: "LaValle, Ch 1"
-  title: "[Course overview, defining VR, historical perspective](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture1-Introduction.pptx)"
+  title: "[Defining VR, course overview, course project expectations](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture1-Introduction.pptx)"
 
 - 
-  date: "Aug 29"
+  date: "Jan 16"
   materials: "LaValle, Ch 2.1, 2.2"
-  title: "[Bird's-eye view: VR hardware](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
+  title: "[Design Processes - Basic Principles and Top Down Design Processes](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
 
 - 
-  date: "Sep 3"
+  date: "Jan 21"
   materials: ~
-  title: "No lecture"
+  title: "**No Lecture: M. L. King Day**"
```

Comment:
There is enough room to put the full name "Martin Luther King Jr. Day". I have never seen this specific abbreviation before.

Reply:
That's the wording on the official UIUC calendar that I copied. Happy to change it though.